packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.routing.js

```diff
@@ -42,6 +42,40 @@ export default /* @ngInject */ ($stateProvider) => {
           Alerter.success(message, 'cloudDB_order_alert');
         }),
       hideBreadcrumb: () => true,
+      onError: /* @ngInject */ ($translate, $anchorScroll) => (error) => {
+        $translate.instant('ovhManagerHostingDatabaseOrderPublic_error', {
```

Comment:
Can you check if this translation key exist in this scope ?

Reply:
does not seem to be used